extras/rya.console/src/test/java/mvm/rya/shell/RyaAdminCommandsTest.java

```diff
@@ -123,6 +124,9 @@ public void deletePCJ() throws InstanceDoesNotExistException, RyaClientException
 
     @Test
     public void getInstanceDetails() throws InstanceDoesNotExistException, RyaClientException {
+        // This test is failed if the default timezone was not EST, so now it's fixed at EST.
+        // If you get assert mismatch of EST!=EDT, try the deprecated getTimeZone("EST") instead.
+        TimeZone.setDefault(TimeZone.getTimeZone("America/New_York"));
```

Comment:
These are two different classes.  Before will still have an order dependence.  
Also every test after will still be affected.
Another solution is to replace it to the original value when the test is over.
With that, only concurrent unit tests will have order dependence, in that case, a race condition.  Then we need to synchronize on the timezone setting.
But what is wrong with setting it everywhere there is a dependence on TimeZone?  All the issues with that all have to do with setting the timezone default globally, but not order, nor concurrency -- unless someone sets it to a non EST timezone, then the concurrency issue comes back, but not the serial order.  

So I think it is best as it is now -- not perfect, but simple.  What do you think?